### PR TITLE
add message id

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

**Review by HackFisher — APPROVED** [review]
LGTM

File: frame/wormhole/backing/s2s/src/lib.rs [hunk 1]
```diff
@@ -99,12 +99,12 @@ pub mod pallet {
 	pub enum Event<T: Config> {
 		/// Token registered [token address, sender]
 		TokenRegistered(Token, AccountId<T>),
-		/// Token locked [token address, sender, recipient, amount]
-		TokenLocked(Token, AccountId<T>, EthereumAddress, U256),
+		/// Token locked [message_id, token address, sender, recipient, amount]
```
Comment (AurevoirXavier) [inline frame/wormhole/backing/s2s/src/lib.rs:102]: use `\[\]` to satisfy rustdoc

**Comment by HackFisher** [comment]
@xiaoch05 Would you mind continue to modify the documents according to https://github.com/darwinia-network/darwinia-common/pull/827, or directly merge from branch xavier-fix2.

Merge one will conflict the other, so better to merge #827 to this PR. cc @AurevoirXavier

**Comment by xiaoch05** [comment]
> @xiaoch05 Would you mind continue to modify the documents according to #827, or directly merge from branch xavier-fix2.
> 
> Merge one will conflict the other, so better to merge #827 to this PR. cc @AurevoirXavier

I  think it's better for me the #827 merged to master first. Then I merge the doc from master.

**Pull request merged**